website/content/docs/upgrading/identity-deduplication.mdx

```diff
@@ -0,0 +1,406 @@
+---
+layout: docs
+page_title: Vault identity duplicate resolution guide
+description: >-
+   Discover whether your Vault cluster contains any identity duplicates and if
+   so, how to safely resolve them.
+---
+
+# Vault identity duplicate resolution guide
+
+Some users may have duplicate identity resources (entities, aliases, groups) in
+their cluster's storage due to historical bugs. These can cause unexpected
+behavior as they are outside of Vault's typical expectations and test scenarios.
+
+Vault 1.19.0 includes additional reporting in server logs to identify if any
+duplicates exist in a cluster. If any do exist, this document provides the
+information necessary to safely resolve them and return the cluster to normal,
+supported behavior.
+
+## Diagnosis
+
+To identify whether your cluster has duplicates you need to follow these steps:
+
+ 1. Ensure you have upgraded to Vault 1.19.0 or higher on all cluster nodes.
+ 2. Check the Vault logs on the active node and locate it's last unseal operation between the following log lines:
+    ```
+    [INFO]  core: post-unseal setup starting
+    ...<check here> ...
+    [INFO]  core: post-unseal setup complete
+    ```
+ 3. If you see the following line, duplicates were found and you should continue
+    to read this guide.
+
+    ```
+    [WARN]  identity: DUPLICATES DETECTED, see following logs for details [...]
+    ```
+
+    **If you don't see this line, you have no further action to take on this
+    cluster**. Ensure you repeat this process on any Performance Replication
+    (PR) secondary clusters though as it's possible for PR secondaries to have
+    local duplicates. Disaster Recovery (DR) secondaries do not need to be
+    checked separately.
+
+## Resolution
+
+If duplicates have been reported in your Vault logs, there are a few steps to
+work through which will depend on the types of duplicates found. Details of each
+are given below. The high-level process for resolution is:
+
+#### With Vault Enterprise Performance or Disaster Recovery replication
+
+ 1. Ensure all clusters (primary and secondaries) are upgraded to 1.19.0 or
+    higher.
+ 1. On the primary, address each duplicate reported using the type-specific
```

Comment:
nit: just because using "type" without a qualifier in the land of computers can be kinda unclear, I might slightly revise this to "... address each reported duplicate using the guidance specific to each duplicate type below"